registry/github/parse.go

```diff
@@ -36,6 +37,12 @@ func (c *client) Parse(path string) (*registry.Source, error) {
 	// nolint: gomnd // ignore magic number
 	parts := strings.SplitN(u.Path, "/", 3)
 
+	// ensure org, repo and filename parts exist
+	// nolint: gomnd // ignore magic number
+	if len(parts) < 3 {
+		return &registry.Source{}, fmt.Errorf("invalid template source %s", path)
```

Comment:
Do you think we should try to augment this error message a bit?

I'm asking because if this change as-is, doesn't really indicate why the template source is invalid.

Thinking that maybe we could try to enrich the message to be a bit more descriptive?

Perhaps something like:

```suggestion
		return &registry.Source{}, fmt.Errorf("template source %s must contain >= 2 '/'", path)
```

or

```suggestion
		return &registry.Source{}, fmt.Errorf("template source %s must contain at least 2 slashes", path)
```

Reply:
If we are going to mention the slashes, I'd rather be clear, such as `("invalid template source %s, must contain org/repo/path-to-filename", path)`. Thoughts?

Reply:
`pathtofilename` isn't elegant, but would hopefully be clear it would cover `org/repo/template.yml` or `org/repo/templates/thistemplate.yml`.

Reply:
Sure 👍 

I'm fine with that approach as well! But I wonder if `filename` is too ambiguous?

What if we did something like this?

```suggestion
		return &registry.Source{}, fmt.Errorf("invalid template source %s: must contain <org>/<repo>/<path_to_template>", path)
```

Reply:
👍  Error has been updated accordingly.